src/index.html

```diff
@@ -1,13 +1,440 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon/favicon.ico"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#" class="header__logo"></a>
+            <a
+              class="icon icon--phone"
+              href="tel:+1 234 5555-55-55">
+              <div class="icon__tooltip">
+                +1 234 5555-55-55
+              </div>
+            </a>
+            <a
+              class="icon icon--menu"
+              href="#menu">
+            </a>
+          </div>
+
+          <div class="header__slogan">
+            <div class="title-label"></div>
+
+            <h1 class="header__title">
+              The world shades.
+              <br>
+              Your music shines.
+            </h1>
+          </div>
```

Comment:
I think this is not a part of the header. I see the next structure here:
- header
  - nav <--- navigation as part of the header
- main
  - section.intro <--- here is a promo image with the slogan
  - sections...

WDYT?

Reply:
As for me, that intro exactly part of header for logical continuation of code structure. 
And slogan couldn't has position in main, because this is another block :)